README.md

```diff
@@ -1,68 +1,102 @@
+- Understand how to sync server data with an applications internal state.
+- Understand how to pass values into a Route component to allow for the updating of state.
+
+## Introduction
+CRUD applications are the foundation of most web applications. Being able to manage creating, edit and deleting data from an external source is as key a skill as it gets. In this project, you will complete the code nessisary to allow all of these fundmental actions.
+
+![Movie DB Example](project-goals.gif)
+
+***Make sure to complete your tasks one at a time and complete test each task before proceeding forward.***
 
 ## Instructions
+### Task 1: Project Set Up
+* [ ] Create a forked copy of this project.
+* [ ] Clone your OWN version of the repository in your terminal
+* [ ] cd into the project base directory `cd web-module-project-reducer-pattern`
+* [ ] Download project dependencies by running `npm install`
+* [ ] Start up the app using `npm start`
+- [ ] With each saved change in your editor, the test runner will re-run all the tests
+- [ ] **IMPORTANT** If a test fails, use the test runner's error messages to find out why it is failing
 
-### Task 1: Project Setup 
+### Task 2: Project Requirements
+#### Editing a Movie
+> *Let's start by walking through the process of adding the routing, component and service calls need for resource updating*
 
-- **Fork** this repository, then clone your fork.
-- Run `npm install` to download dependencies.
-- Run the server using `npm start`.
-- In a separate terminal cd into the `client` folder and run `npm install` to download dependencies.
-- Still inside the `client` folder run `npm start` to run the client application.
+* [ ] First, we need to be able to navigate to the edit movie component. Add in the premade component used to edit a movie.
 
-### Task 2: MVP
+* [ ] Next, we need to be able to load in the current movie's attributes into our local form state. Add in the code needed to retrieve the movie data of the passed in id when the component mounts.
 
-#### Updating A Movie:
+* [ ] At this point, nothing happens when the edit form is submitted. Add in the service call need to update the server with our updated movie when the edit form is submit button is clicked.
 
-- Add a route at the path `/update-movie/:id`
-- Create a component with a form to update the chosen movie
-- Add a button in the movie component that routes you to your new route with the movies's id as the URL param
-- The form should make a PUT request to the server when submitted
-- When the call comes back successfully, reset your form state and route the user to `/` where they will see the updated movie in the list
-- (Editing the movie stars can be a stretch problem... don't worry about it now. Move on to the next step, and come back and solve this at the end)
+* [ ] Don't forget to make sure that your server data and your local state are in sync! Make any changes the edit route needed to modify global state within the edit form.
```

Comment:
I would suggest some minor tweaks here -

"Don't forget to make sure that your server data and your local state are in sync! Make any changes to the edit component needed to modify the global state from within the edit form."